SimG4Core/MagneticField/src/G4TMagFieldEquationCMS.h

```diff
@@ -0,0 +1,88 @@
+//
+// ********************************************************************
```

Comment:
G4 disclaimer better to remove.

Reply:
Are we sure that we are authorized to copy a G4 file and remove the discaimer?

Reply:
there was a campaign few years ago to do this, because CMSSW should not distribute original Geant4 code. In the header, it is possible to say that it is a CMS modification of the original G4TDormandPrince45. 

We have several modified Geant4 classes - it is not in contradiction with Geant4 license. By the way, with time they may be very different from the original. In several cases (not in all) I replace "G4" to "CMS" in class names, however, few remains, likely the campaign should be completed. In general, it is more clear if classes from CMSSW do not start from "G4".